go-controller/pkg/crd/userdefinednetwork/v1/cudn.go

```diff
@@ -218,3 +234,30 @@ type VLANConfig struct {
 	// +optional
 	Access *AccessVLANConfig `json:"access"`
 }
+
+type TransportOption string
+type SNATOption string
+type RoutingOption string
+
+const (
+	TransportOptionNoOverlay TransportOption = "NoOverlay"
+	TransportOptionGeneve    TransportOption = "Geneve"
```

Comment:
I was rebasing the evpn api change on top of this and realised I was automatically doing "EVPN" here instead of "Evpn" like you have for Geneve which is an acronym.. I guess its ok cause I don't really want to shout out GENEVE.